guides/request-validation-sinatra/example-1/example-1.3.x.rb

```diff
@@ -0,0 +1,17 @@
+require 'sinatra'
```

Comment:
A good approach is to introduce an explanatory commentary, similar to the ones we already introduced to the other code snippets. Did you consider adding a comment like this:

```
// You can find your Twilio Auth Token here: https://www.twilio.com/console
// Set at runtime as follows:
// $ TWILIO_AUTH_TOKEN=XXXXXXXXXXXXXXXXXXX node index.js
//
// This will not work unless you export the TWILIO_AUTH_TOKEN environment variable.
//
```